lib/bgs/form686c.rb

```diff
@@ -96,8 +97,26 @@ def get_state_type(payload)
       'Started'
     end
 
+    # if we are setting the claim to be manually reviewed
+    # and the Veteran is currently receiving pension benefits
+    # set the claim type code to 130DAEBNPMCR (PMC eBenefits Dependency Adjustment Reject)
+    # else use 130DPNEBNADJ (eBenefits Dependency Adjustment)
+    def get_claim_type(proc_state)
+      if proc_state == 'MANUAL_VAGOV'
```

Comment:
Although this may be a widely used pattern in the repo (and outside the scope of this PR), should this string be a constant somewhere? Something like:
```suggestion
      if proc_state == BGS::ProcStates::MANUAL_VAGOV
```